### ISSUE TITLE
Exception thrown for ontology fields

### ISSUE THREAD

**Issue by lsgs** [lsgs/redcap-extended-reports#2]
Exception thrown for ontology fields in report view and export.

> Next Exception: An error occurred while running an External Module query: 'Statement preparation failed'. The error from the database was: 'You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '\REDCap.redcap_web_service_cache where project_id=? and service=? and categor...' at line 1'. See the server error log for more details. in ...

```
ExternalModules\AbstractExternalModule->query()
#6 /site/redcap/modules/extended_reports_v2.0.0/Report.php(1061): MCRI\ExtendedReports\Report->makeOntologyDisplay()
```

**Comment by lsgs** [comment]
Note also in this case (pid=9/report=72779) no reshaping is actually required because the project is not longitudinal and the report contains no fields from repeating instruments.

**Comment by lsgs** [comment]
Fixed in release 2.0.1
